src/Arcus.Scripting.KeyVault/Scripts/Get-AzKeyVaultAccessPolicies.ps1

```diff
@@ -15,11 +15,11 @@ else
 	$keyVault = Get-AzKeyVault -VaultName $keyVaultName -ResourceGroupName $resourceGroupName
 }
 
-if($keyVault)
-{
-	$armAccessPolicies = @()
-	$keyVaultAccessPolicies = $keyVault.accessPolicies
+$armAccessPolicies = @()
+$keyVaultAccessPolicies = $keyVault.accessPolicies
```

Comment:
Any specific reason why you're setting the $keyVaultAccessPolicies-variable outside of the if-statement?
This will only be used within that statement and will remain empty when the $keyVault-variable remains null.

Reply:
No, should indeed be put inside the if.  I'll change that.